webapp/src/components/Navbar.jsx

```diff
@@ -38,6 +39,9 @@ function Navbar() {
                         <li>
                             <Link class="block py-2 px-3 text-gray-900 rounded hover:bg-gray-100 md:hover:bg-transparent md:hover:text-blue-700 md:p-0 md:dark:hover:text-blue-500 dark:text-white dark:hover:bg-gray-700 dark:hover:text-white md:dark:hover:bg-transparent dark:border-gray-700" to="/rankings">Rankings</Link>
                         </li>
+                        {isAuthenticated() && (
```

Comment:
Please don't use  && in async checks as i tcan leads to problems it's better to use ?